src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name 'AreaCalculator' is a noun, which is good, but since interfaces are not meant to be calculators themselves but rather to provide a contract for calculating, consider naming it something like 'AreaCalculable' to indicate that implementing classes can calculate an area.